src/lib/components/ContentfulRichText/ContentfulRichText.test.ts

```diff
@@ -1,14 +1,14 @@
 import "@testing-library/jest-dom";
 import { render } from "@testing-library/svelte";
 import { describe, it, expect } from "vitest";
-import "$lib/__tests__/extendExpect.ts";
+import "$lib/__tests__/extendExpect";
```

Comment:
Removing the `.ts` resolved a `rollup` warning we were getting in the build logs.